### ISSUE TITLE
not and greaterThan should work in case insensitive manner

### ISSUE THREAD

**Issue by sailsbot** [balderdashy/sails#4021]
_From @dmarcelino on April 30, 2015 10:41_

In balderdashy/waterline-adapter-tests#53 I've added case insensitive `not` and `greaterThan` tests but unfortunately sails-mongo is breaking those tests: https://travis-ci.org/balderdashy/waterline-adapter-tests/jobs/60676152#L542

PR #235 looked like a potential fix but after trying it the tests still fail.

We need to fix this issue to put sails-mongo on par with the other adapters.

Perhaps http://stackoverflow.com/questions/1863399/mongodb-is-it-possible-to-make-a-case-insensitive-query can help us get here...

cc: @andypham, @particlebanana, @tjwebb, @devinivy, @andyhu, @mgenev

_Copied from original issue: balderdashy/sails-mongo#265_

**Comment by sailsbot** [comment]
Thanks for posting, @dmarcelino.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:
- review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
- create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by particlebanana** [comment]
Solved and now mongo is case-sensitive in Waterline 0.13 so wrapping this one up.
